ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -343,13 +375,19 @@ public View getBannerView() {
         return adLayout;
     }
 
-    private boolean hasBannerSizeAd(AdSize adSize) {
-        if (300 == adSize.getWidth() && 250 == adSize.getHeight()) {
-            adConfig.setAdSize(VUNGLE_MREC);
-            return true;
+    private AdConfig.AdSize getVungleAdSize(AdSize adSize) {
+        AdConfig.AdSize adSizeType = null;
+
+        if (VUNGLE_MREC.getWidth() == adSize.getWidth() && VUNGLE_MREC.getHeight() == adSize.getHeight()) {

```

Comment:
@HuFangshuai I can't find any dimensions in com.google.android.gms.ad.AdSize for Banner short btw. Is this why we can't do 

```
        if (adSize == AdSize.BANNER) {
            adSizeType = BANNER;
        } else if (adSize == AdSize.MEDIUM_RECTANGLE) {
```